pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -211,19 +211,12 @@ func (r *ReconcileMigCluster) Reconcile(request reconcile.Request) (reconcile.Re
 
 // Setup remote watch.
 func (r *ReconcileMigCluster) setupRemoteWatch(cluster *migapi.MigCluster) error {
+	var err error
 	nsName := types.NamespacedName{
 		Namespace: cluster.Namespace,
 		Name:      cluster.Name,
 	}
-	remoteWatchMap := remote.GetWatchMap()
-	remoteWatchCluster := remoteWatchMap.Get(nsName)
```

Comment:
We're removing these here, but I don't see the underlying functions being removed. Are these functions no longer needed?

Reply:
I found it below.